pages/api/patients/create.ts

```diff
@@ -0,0 +1,48 @@
+// Next.js API route support: https://nextjs.org/docs/api-routes/introduction
+import type { NextApiRequest, NextApiResponse } from 'next'
+import prisma from '../../../lib/prisma'
+
+export default async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse
+) {
+  if (req.method === 'POST') {
+    try {
+      let { user_id, pickup_enabled, dob, prescriptions } = req.body.data
+
+      var patient = null
```

Comment:
avoid the use of `var` since it can be accessed outside the current function context
```suggestion
      let patient = null
```